test/irmin-pack/test_upgrade.ml

```diff
@@ -86,7 +90,7 @@ let key_of_entry x =
 type start_mode = From_v2 | From_v3 | From_scratch [@@deriving irmin]
 
 type setup = {
-  indexing_strategy : [ `always | `minimal ];
+  indexing_strategy : [ `always | `minimal | `minimal_gc ];
```

Comment:
`minimal_gc` is not an indexing strategy 😛 

If I understand correctly `archive_v3_minimal_gced` is a store where a GC was run targeting `c0`.

I suggest you alter `start_mode` instead of `indexing_strategy`. You could name the new tag something like `From_v3_c0_gced`.

Reply:
That's better, done